### DISCUSSION TITLE
Export mode unavailable - Help?

### DISCUSSION THREAD

**Discussion by heyoitscelo** [mifi/lossless-cut#2194] [Support]
Hey brother, I'm on Mac (Apple M1),

I was able to export 1 file yesterday. After that, when I try to export, it exports the files separately as a document. I don't know what happened, but I loved the way it worked before it started exporting the files as documents. 

I'll post the part where it shows next to the Output files and the segments to export. I've already uninstalled and installed the app a few times to see if it would reset, though it didn't.

What can I do to recover the export mode where I can export as .mov or mp4?

![Lossless cut](https://github.com/user-attachments/a
![Lossless cut Export options](https://github.com/user-attachments/assets/ff89e412-2624-4218-8893-21e7cd46b957)
ssets/022f6d86-8383-4277-bf53-038a3e27897d)

**Comment by mifi** [comment]
how many segments do you have? you can select export mode "Merge cuts" only when you have more than one (selected) segment. you acn select segments in the right segment panel

**Reply by heyoitscelo** [reply]
I want to export the edits as .mov or .mp4 files, although they are being exported as text files. 
I apologize about the music, I didn't realize OBS was capturing it.

As you can see in the video, the clip gets edited and comes out as a text file, and I don't know why. I also don't understand anything about coding, to be honest.

https://github.com/user-attachments/assets/b13c7c0c-a68a-4de3-818a-fca7f75707eb
